packages/cli/src/create/command.ts

```diff
@@ -97,185 +107,178 @@ export async function twCreate(
     projectType = "app";
   }
 
-  if (!projectPath) {
-    const defaultName =
-      projectType === "contract" ? "thirdweb-contracts" : "thirdweb-app";
-    const res = await prompts({
-      type: "text",
-      name: "path",
-      message: CREATE_MESSAGES.projectName,
-      initial: options.template || defaultName,
-      format: (name: string) => name.toLowerCase(),
-      validate: (name: string) => {
-        const validation = validateNpmName(
-          path.basename(path.resolve(name.toLowerCase())),
-        );
-        if (validation.valid) {
-          return true;
-        }
-        return "Invalid project name: " + validation.problems?.[0];
-      },
-    });
-
-    if (typeof res.path === "string") {
-      projectPath = res.path.trim();
+  let onlyContract = false;
+  // Whether to only create a new contract without the project
+  const contractProjectType = await detect(projectPath, {});
```

Comment:
wouldn't it be safer to rely on this being one of the contract project types we support instead of checking if there's a `contracts` folder in the current dir?

Reply:
yeah can prob use detector for this

Reply:
The detector is being used above - but probably to get the correct folder, instead of expecting src/contracts, we should just iterate through folders and check for one with a `.sol` file - and then afterwards default to src or contracts